src/integration_tests/algorithmic_style_test.py

```diff
@@ -0,0 +1,79 @@
+"""End-to-end test cases of algorithmic style."""
+
+from __future__ import annotations
+
+from typing import Any, Callable
+
+from latexify import frontend
```

Comment:
I can move it, but `function_expansion_test` and `regression_test` basically use `frontend` directly, as they use `frontend.function`.

Similarly, this file uses `function.get_latex`. Both of these functions are exported in `__init__.py`, so it feels somewhat like an integration test, because we're testing end-to-end behavior from function to generated $LaTeX$ output

Reply:
So it turns that the issue comes from the old implementation of these tests, will consider it later.